include_core/AtomicSupport.hpp

```diff
@@ -312,7 +312,7 @@ class VM_AtomicSupport
 #elif defined(ARM) /* defined(J9HAMMER) */
 		__sync_synchronize();
 #elif defined(AARCH64) /* defined(ARM) */
-		__asm __volatile ("dmb ishld":::"memory")
+		__asm __volatile ("dmb ishld":::"memory");
```

Comment:
I think these^^ changes belong in a separate commit, it's totally unrelated to CI.

Reply:
Ok. I'll split them out in another PR. They were inadvertently missed in the Build and/or Part PR(s).

Reply:
This was moved to and merged in PR #2549.